qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -860,7 +858,12 @@ def Ud(a, b, c):
 
 
 def trace_to_fid(trace):
-    r"""Average gate fidelity is :math:`Fbar = (d + |Tr (U_{target} \cdot U^{\dag})|^2) / d(d+1)`
+    r"""Average gate fidelity is
+
+    .. math::
+
+        \bar{F} = \frac{d + |Tr (U_\text{target} \cdot U^{\dag})|^2}{d(d+1)}
```

Comment:
Just a suggestion, but I think putting the trace in normal font looks much nicer 🙂 
```suggestion
        \bar{F} = \frac{d + |\mathrm{Tr} (U_\text{target} \cdot U^{\dag})|^2}{d(d+1)}
```